CMakeLists.txt

```diff
@@ -128,6 +128,13 @@ else()
   endif()
   list(APPEND COMPUTECPP_USER_FLAGS "-Xclang -cl-mad-enable")
   list(APPEND COMPUTECPP_USER_FLAGS "-Xclang -cl-no-signed-zeros")
+
+  if (MSVC)
+    # Allows compute++ to parse the Windows headers
+    list(APPEND COMPUTECPP_USER_FLAGS "-D_ALLOW_COMPILER_AND_STL_VERSION_MISMATCH=1")    
```

Comment:
1. Can you strip the trailing whitespace please?
2. Is this best here or in FindComputeCpp?

Reply:
This problem has been hit before so there's a module in the SDK which does more-or-less the same thing, but with extra logic to try compiling stuff so a more useful error can be reported. It's not in `FindComputeCpp.cmake` but we could put it in there directly if you like.

https://github.com/codeplaysoftware/computecpp-sdk/blob/master/cmake/Modules/ComputeCppCompilerChecks.cmake

Reply:
This was moved to the sycl-dnn's FindComputeCpp since it has sycl-dnn specific code.